planning/behavior_path_planner/src/scene_module/lane_change/util.cpp

```diff
@@ -128,8 +128,15 @@ std::optional<LaneChangePath> constructCandidatePath(
 
   LaneChangePath candidate_path;
   candidate_path.acceleration = acceleration;
-  candidate_path.preparation_length = prepare_distance;
-  candidate_path.lane_change_length = lane_change_distance;
+  candidate_path.length.prepare = prepare_distance;
+  candidate_path.length.lane_changing = lane_change_distance;
+  candidate_path.duration.prepare = std::invoke([&]() {
+    const auto duration =
+      prepare_distance / std::max(lane_change_param.minimum_lane_change_velocity, speed.prepare);
+    return std::min(duration, lane_change_param.lane_change_prepare_duration);
+  });
+  candidate_path.duration.lane_changing =
+    std::round((lane_change_distance / lane_changing_speed) * 2.0) / 2.0;
```

Comment:
It seems better to change like following:
```
std::ceil((lane_change_distance / lane_changing_speed) / resolution) * resolution;
```

Reply:
Fixed in [7471723](https://github.com/autowarefoundation/autoware.universe/pull/2704/commits/747172397e70404294b019fb1999797b39286848)